ansible/playbooks/send-message.yaml

```diff
@@ -40,6 +40,7 @@
     - set_fact:
         dynamodb_tablename: "{{ stack_manager_stack_outputs.AemStackManagerTableName }}"
         sns_topic: "{{ stack_manager_stack_outputs.StackManagerTopicArn }}"
+        externalId: "{{ 9999999999999999999999 | random | to_uuid }}"
```

Comment:
Please rename externalId to eventId, and then allow the user to inject the eventId (e.g. users can then pass in CI tool's build ID as the event ID).
Also ensure that the event ID shows up (log/debug/etc) at the end of the output so user can easily identify what the event ID is, which could then be used for querying command executions using the event ID. This ID can stay as random uuid when the users don't supply an event ID, alternatively, time in milliseconds is also fine.
Just as a note from previous discussion, the benefit of having event ID is to allow the ability to query the commands based on user-generated ID, so that the users don't have to check the AWS-generated command ID.